### PR TITLE
SSCSSI-425: update case to V2 on hmctsResponseReviewed event

### PR DESCRIPTION
**Pull request by SomeshRA**
### Jira link
https://tools.hmcts.net/jira/browse/SSCSSI-425

### Change description
update case to V2 on hmctsResponseReviewed event

### Checklist

- [ x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedSubmittedHandler.java [hunk 1]
```diff
@@ -53,24 +57,40 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
 
         if (sscsCaseData.getIsInterlocRequired() != null && sscsCaseData.getIsInterlocRequired().equals("Yes")) {
             String whoToReview = sscsCaseData.getSelectWhoReviewsCase().getValue().getCode().equals("reviewByJudge") ? "Judge" : "TCW";
-            updateCase(sscsCaseData, callback.getCaseDetails().getId(), EventType.VALID_SEND_TO_INTERLOC, "Send to interloc", "Send a case to a " + whoToReview + " for review");
+            updateCase(
```
Comment (sanjayparekh-hmcts) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedSubmittedHandler.java:60]: I think this call first is in effect just a triggerEventV2

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedSubmittedHandler.java (hunk 1 above)
Comment (SomeshRA) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedSubmittedHandler.java:60]: Yeah it should be. Had to keep it generic since its a common code - updateCase .

Comment (sanjayparekh-hmcts) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedSubmittedHandler.java:60]: Actually, there's no real reason for it to be generic. It might be better to split out the 2 types of calls into their own methods. Might be a bit clearer for devs in the long-term

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]

**Review by sanjayparekh-hmcts — DISMISSED** [review]

**Review by srochani — DISMISSED** [review]

File: charts/sscs-tribunals-api/values.preview.template.yaml [hunk 2]
```diff
@@ -65,10 +65,11 @@ java:
     HOURS_END_TIME: 23
     RUN_DB_MIGRATION_ON_STARTUP: true
     TEST_RECIPIENTS_POSTCODE: "*"
-    INFECTED_BLOOD_COMPENSATION_ENABLED: true
+    INFECTED_BLOOD_COMPENSATION_ENABLED: false
     SUBMIT_APPEAL_SERVICE_V2_ENABLED: true
     PROCESS_EVENT_MESSAGE_V2_ENABLED: true
     UPDATE_CASE_ONLY_HEARING_V2_ENABLED: true
+    TRIGGER_EVENT_V2_ENABLED: true
     # START - SSCS-IB RELEASE VARIABLES
```
Comment (sanjayparekh-hmcts) [inline charts/sscs-tribunals-api/values.preview.template.yaml:73]: why did you need to enable this?

Comment (SomeshRA) [reply charts/sscs-tribunals-api/values.preview.template.yaml:73]: This was added to enable AppealReceivedHandler to take V2 path - triggerCaseEventV2

Comment (SomeshRA) [reply charts/sscs-tribunals-api/values.preview.template.yaml]: As discussed enabled it on values.yaml

**Review by sanjayparekh-hmcts — APPROVED** [review]
